waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -55,10 +56,11 @@ preCleanup :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO ()
 preCleanup spec projectRootDir = do
   deleteGeneratedMigrationsDirIfRedundant spec projectRootDir
 
--- * Db generator
-
-genDb :: AppSpec -> [FileDraft]
-genDb spec = genPrismaSchema spec : maybeToList (genMigrationsDir spec)
+genDb :: AppSpec -> Generator [FileDraft]
+genDb spec = do
+  prismaSchema <- genPrismaSchema spec
+  maybeMigrationsDir <- genMigrationsDir spec
+  return $ prismaSchema : maybeToList maybeMigrationsDir
```

Comment:
My feeble Haskell foo does find that a bit harder to grok 🤯 , but I am impressed it is a one liner! ⛳

Reply:
If something can be put into one line it doesn't mean it should though :D (and this is good example).

Reply:
Btw if you added `<:>` operator, then this could be:
```hs
genPrismaSchema spec <:> (maybeToList <$> genMigrationsDir spec)
```
at least I think so, haven't tried it out.

Reply:
Good point, I will tinker once I add `<:>`, thanks

Reply:
That worked, thanks!

```haskell
genDb :: AppSpec -> Generator [FileDraft]
genDb spec =
  genPrismaSchema spec <:> (maybeToList <$> genMigrationsDir spec)
```